tests/integration/folder-option-spec.js

```diff
@@ -62,7 +62,7 @@ test.describeIntegrationTest("Outputting to another directory:", function() {
             test.assert.verifyBundleIs(
                 ';var file1="file1";\n' +
                 ';var file2="file2";\n' +
-                ';window.JST=window.JST||{},JST.file3=new Hogan.Template({code:function(i,n,e){var a=this;return a.b(e=e||""),a.b("<div> "),a.b(a.v(a.f("c",i,n,0))),a.b(" </div>"),a.fl()},partials:{},subs:{}});\n'
+                ';(function(){vare,i,t="undefined"!=typeofmodule&&module.exports;e=t?require("hogan"):this.Hogan,i=newe.Template({code:function(e,i,t){varo=this;returno.b(t=t||""),o.b("<div>"),o.b(o.v(o.f("c",e,i,0))),o.b("</div>"),o.fl()},partials:{},subs:{}}),t?module.exports=i:(this.JST=this.JST||{},JST.file3=i)}).call(this);\n'

```

Comment:
I don't know much about what this is but `i=newe.Template` seems wrong. Shouldn't it be `i=new e.Template`?